api/apps/api/src/modules/access-control/projects-acl/project-acl.controller.ts

```diff
@@ -0,0 +1,80 @@
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import {
+  Controller,
+  Get,
+  Req,
+  Param,
+  ParseUUIDPipe,
+  UseGuards,
+  ForbiddenException,
+  Delete,
+  Patch,
+  Body,
+} from '@nestjs/common';
+import { JwtAuthGuard } from '@marxan-api/guards/jwt-auth.guard';
+import { ProjectAclService } from './project-acl.service';
+import { RequestWithAuthenticatedUser } from '@marxan-api/app.controller';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { UsersProjectsApiEntity } from '@marxan-api/modules/projects/control-level/users-projects.api.entity';
+import { UpdateUserRoleinProject } from './dto/update-user-role-project.dto';
+
+@UseGuards(JwtAuthGuard)
+@ApiBearerAuth()
+@ApiTags('Projects Users')
+@Controller(`${apiGlobalPrefixes.v1}/projects`)
+export class ProjectAclController {
+  constructor(private readonly projectAclService: ProjectAclService) {}
+
+  @Get(':projectId/users')
+  async findUsersInProject(
+    @Param('projectId', ParseUUIDPipe) projectId: string,
+    @Req() req: RequestWithAuthenticatedUser,
+  ): Promise<UsersProjectsApiEntity[]> {
+    const isOwner = await this.projectAclService.checkUserIsOwner(
+      req.user.id,
+      projectId,
+    );
```

Comment:
Seems like `controller` is keeping some logic (check Acl, find something) which would be great to move to the service.

Controller is only one of the ways to access the feature. If any other module would like to get the usersIds within a project, would it need to use it through controller? otherwise, it would skip the "check" the module is developed for.